pkg/gofr/http/router.go

```diff
@@ -56,7 +59,12 @@ func (rou *Router) AddStaticFiles(endpoint, dirName string) {
 	cfg := staticFileConfig{directoryName: dirName}
 
 	fileServer := http.FileServer(http.Dir(cfg.directoryName))
-	rou.Router.NewRoute().PathPrefix(endpoint + "/").Handler(http.StripPrefix(endpoint, cfg.staticHandler(fileServer)))
+
+	if endpoint == "/" {
+		rou.Router.NewRoute().PathPrefix("/").Handler(cfg.staticHandler(fileServer))
+	} else {
+		rou.Router.NewRoute().PathPrefix(endpoint + "/").Handler(http.StripPrefix(endpoint, cfg.staticHandler(fileServer)))
+	}
```

Comment:
I'm unsure it's the right place (lines of code or even the right PR) to comment this, but I'm commenting here, when I think about it.

If my inputs lead to a conclusion, please consider opening an issue.

So here is my point:

Are we sure the URL path is cleaned?

I found many framework having issues when a path in URL was provided with `//`

Like `//favicon.ico` or `/.well-known//health` it's often caused by a URL of the server containing a trailing slash in a environment variable.

This often leads to 404, while an Apache2 or NGinx server would have served the content

My remark is valid for static files but also for API endpoints

Reply:
Sure will add that issue